mappings/net/minecraft/world/chunk/ChunkState.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_whqcauvq net/minecraft/world/chunk/ChunkState
+	FIELD f_fajrjukk FULL Lnet/minecraft/unmapped/C_whqcauvq;
+	FIELD f_qnqyfpms INACCESSIBLE Lnet/minecraft/unmapped/C_whqcauvq;
+	FIELD f_temaefsx BLOCK_TICKING Lnet/minecraft/unmapped/C_whqcauvq;
+	FIELD f_ymzbvdja ENTITY_TICKING Lnet/minecraft/unmapped/C_whqcauvq;
+	METHOD m_rvjdcoiw isStateHigherOrEqual (Lnet/minecraft/unmapped/C_whqcauvq;)Z
```

Comment:
hm, I think there's potential for a better name here but I'm not sure what
maybe something similar to moj, `matchesOrAfter`?